PyRDF/Backend/Dist.py

```diff
@@ -118,17 +119,8 @@ class Dist(Backend.Backend):
             information about any friend trees of the main ROOT.TTree
     """
 
-    def __init__(self, config={}):
```

Comment:
I wonder if it makes sense that both `Backend` and `Dist` exist, now that all backends are going to be distributed. We can still keep the hierarchy if we think it makes sense to make that distinction based on the content of these modules. Is it going to be now something like `BaseBackend` (with some basic functions) and `DistBackend` with some functions more specific to distribution?

Reply:
I agree better to keep them separate for readability and clarity. Also the proposed names sound good